src/tools/sdk/_windows.md

```diff
@@ -14,10 +16,33 @@ To install the Dart SDK:
 C:\> choco install dart-sdk
 ```
 
-To **upgrade** the Dart SDK:
+To upgrade the Dart SDK:
 
 ```terminal
 C:\> choco upgrade dart-sdk
 ```
 
+Default installation directory:
+
+```terminal
+C:\tools\dart-sdk
+```
+
+How to change installation directory?
+
+This package uses the help Get-BinRoot, which means that 
+you can set an environment variable called ChocolateyToolsLocation, which 
+controls where this package is extracted to. 
+For more information check out [get-toolslocation].
+
+How to set path?
```

Comment:
As a clarifying question before I review this section further, does choco not automatically add packages to the path?

Reply:
It does, but for some cases (as reported in the original issue) it was not being added. So I added this information as well just in case someone struggles with it. But yes, we can put this under a troubleshooting heading.

Reply:
Thanks for including this then, it will definitely be helpful. 

Instead of starting this section with a question, how about we switch to some sort of troubleshooting statement like you mentioned. I'm not exactly sure how to structure or state this, do you have any ideas? It could be something basic like:

```suggestion
If you don't have access to the Dart executables,
add the extraction location to your PATH:
```

Reply:
Yes, looks cleaner now.